source/hic_hal/nxp/lpc11u35/uart.c

```diff
@@ -101,6 +101,10 @@ int32_t uart_set_configuration(UART_Configuration *config)
     baudrate = config->Baudrate;
     // Compute baud rate dividers
     mv = 15;
+#ifdef BOARD_RTL8195AM
+    // Due to hardware issue, RTL8195AM's main cpu can only talk 38400 with DAP UART.
+    config->Baudrate = 38400;
```

Comment:
Followup on the errata, or data sheet explaining this baud rate.

usbd_cdc_acm.c has some _weak functions you can override for the initialization of the line coding instead hardcoded value in this file.

Reply:
Thanks Brian @brianesquilona . What action do we need from Realtek team to continue? Loop @M-ichae-l @ARMmbed/team-realtek

Reply:
Thanks!
I believe that the <Verify flash algo function> needs to be added to DAPLink.

Reply:
Verify function api will be added to Daplink, but Realtek has to implement/export this to the flash algo and point the verify address, this code be done after I change the code.

But this fixing of baud can be done independently and should changed to implementing the weak functions in  usbd_cdc_acm.c

Reply:
Hi @M-ichae-l , please follow Brian's suggestion to use the weak functions in usbd_cdc_acm to fix the baud rate instead of hard coding it. Thanks.

Hi @brianesquilona , please let us know when you have Verify api added so Realtek can proceed implementing to the flash algo and point the verify address. Thanks.

Reply:
#491 
This is the PR which includes verify api to DapLink

Reply:
Thanks @brianesquilona 
Hi @M-ichae-l , please proceed and let us know if you have questions. Thanks.

Reply:
Hi @M-ichae-l , any progress of adding the implementation of verify API on top of #491 and modifying the UART with usbd_cdc_acm function based on Brian's comments? Thanks

Reply:
@samchuarm 
I have finished the verify function updates, working on the weak functions in usbd_cdc_acm.c. Should we put the two updates in one PR  in the future?